<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drone Control GUI with Serial Connection</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftPanel">
       <layout class="QVBoxLayout" name="leftPanelLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="drone3dLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="drone2dLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="slidersLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="scaleSliderLayout">
            <item>
             <widget class="QLabel" name="scaleLabel">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: purple; font-size: 11px;</string>
              </property>
              <property name="text">
               <string>Scale: 1.0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="scaleSlider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="xoffSliderLayout">
            <item>
             <widget class="QLabel" name="xoffLabel">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: red; font-size: 11px;</string>
              </property>
              <property name="text">
               <string>X offset: 0.0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="xoffSlider">
              <property name="minimum">
               <number>-50</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="yoffSliderLayout">
            <item>
             <widget class="QLabel" name="yoffLabel">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: green; font-size: 11px;</string>
              </property>
              <property name="text">
               <string>Y offset: 0.0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="yoffSlider">
              <property name="minimum">
               <number>-50</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="zoffSliderLayout">
            <item>
             <widget class="QLabel" name="zoffLabel">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: blue; font-size: 11px;</string>
              </property>
              <property name="text">
               <string>Z offset: 0.0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="zoffSlider">
              <property name="minimum">
               <number>-50</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightPanel">
       <layout class="QVBoxLayout" name="rightPanelLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="actuatorCanvasLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="selectionBarLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="createBranchButton">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Create Branch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openEventDesignerButton">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Open Waveform Designer GUI</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openHapticPatternButton">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Open Haptic Pattern GUI</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="eventLibraryLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="serialConnectionLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
